packages/database/schema.prisma

```diff
@@ -34,6 +34,7 @@ model Webhook {
   createdAt     DateTime           @default(now()) @map(name: "created_at")
   updatedAt     DateTime           @updatedAt @map(name: "updated_at")
   url           String
+  source        String?            @default("user")
```

Comment:
can we make the source an enum with values `"user" | "zapier"`

Reply:
okay, this makes sense! sorry for missing this